README.md

```diff
@@ -293,14 +293,20 @@ However, it does not aim to implement the whole specification; in particular it
 
 The X.509 and TLS code can use PSA cryptography for most operations. To enable this support, activate the compilation option `MBEDTLS_USE_PSA_CRYPTO` in `mbedtls_config.h`. Note that TLS 1.3 uses PSA cryptography for most operations regardless of this option. See `docs/use-psa-crypto.md` for details.
 
-### Upcoming features
+### PSA drivers
 
-Future releases of this library will include:
+Mbed TLS supports drivers for cryptographic accelerators, secure elements and random generators. This is work in progress. Please note that the driver interfaces are not fully stable yet and may change without notice. We intend to preserve backward compatibility for application code (using the PSA Crypto API), but the code of the drivers may have to change in future minor releases of Mbed TLS.
 
```

Comment:
Also link `docs/psa-driver-example-and-guide.md` from https://github.com/Mbed-TLS/mbedtls/pull/6915 once that is merged. Note that unlike the others, this one discusses the present of Mbed TLS, not the future.

Reply:
It's now merged.